test-tools/setup.ts

```diff
@@ -0,0 +1,8 @@
+import { chai, sinon } from './utils';
```

Comment:
The files in this directory look the same as the one for the logic repo. We should consider consolidating these files into the devtools repo the future.

Reply:
Will make a follow up ticket to get that done in the future.